src/libsyntax/parse/parser.rs

```diff
@@ -2052,9 +2052,23 @@ impl<'a> Parser<'a> {
                 while self.token != token::CloseDelim(token::Paren) {
```

Comment:
This arm should probably be rewritten using the standard list parsing infra (could try that as future work...).

Reply:
If you have a sec, can you file an issue for this? I might fix it myself but the reminder would be great. ^^

Reply:
https://github.com/rust-lang/rust/issues/63396